apps/editor/src/app/atoms/dashboard/subscriberChart.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useMemo} from 'react'
+import {useTranslation} from 'react-i18next'
+import {BarChart, Bar, XAxis, YAxis, ResponsiveContainer} from 'recharts'
+import styled from '@emotion/styled'
+import {useNewSubscribersPerMonthQuery} from '@wepublish/editor/api'
+import {SubscribersPerMonth} from '@wepublish/api'
+
+const BarChartErrorText = styled.p`
+  font-size: 1rem;
+  font-style: italic;
+  text-align: center;
+  margin-top: 2rem;
+`
+
+export function SubscriberChart() {
+  const {t} = useTranslation()
+  const {data, loading} = useNewSubscribersPerMonthQuery({variables: {months: 12}})
+
+  const pastYearSubscriptions = useMemo(() => data?.newSubscribersPerMonth ?? [], [data])
```

Comment:
```suggestion
  const pastYearSubscriptions = data?.newSubscribersPerMonth ?? []
```

Reply:
no useMemo needed as the CPU impact is super minimal but the memory impact is a lot bigger with useMemo